### ISSUE TITLE
Career Service content accessibility audit

### ISSUE THREAD

**Issue by Computerrat** [phsc-edu/2018#342]
### Please check the following links for accessibility problems. ###

---------------

https://career-services.phsc.edu/help/faq/how-can-i-find-out-more-about-testing-services-offered-phsc
![how can i find out more about testing services offered at phsc_ _ career services - google chrome 2018-08-06 14 28 32](https://user-images.githubusercontent.com/31025634/43847441-9baaad10-9afe-11e8-9999-42ec8d477a79.png)

-----------------

https://career-services.phsc.edu/help/step-step-instructions/how-do-i-register-phsc-job-board-college-central-network
![how do i register for the phsc job board college central network_ _ career services - google chrome 2018-08-06 14 29 51](https://user-images.githubusercontent.com/31025634/43847516-c59ee816-9afe-11e8-890e-99c4a83d07b2.png)

-----------------

https://career-services.phsc.edu/help/step-step-instructions/what-are-instructions-how-to-create-focus-2-account-and-select-top
![what are the instructions on how to create a focus 2 account and select a _top choice_ major_ _ career services - google chrome 2018-08-06 14 38 45](https://user-images.githubusercontent.com/31025634/43847609-fda55f10-9afe-11e8-870e-06baa1849aa9.png)

---------------------

https://career-services.phsc.edu/help/faq/internships/how-do-i-register-drafting-paralegal-criminal-justice-or-human-services
![how do i register for a drafting paralegal criminal justice or human services internship_ _ career services - google chrome 2018-08-06 14 41 49](https://user-images.githubusercontent.com/31025634/43847667-237684da-9aff-11e8-8bf4-5e0aa0b74b0d.png)

**Comment by C13L0** [comment]
@curtiskphsc @boettcc Please correct the accessibility issues as documented above, along with the missing links.

**Comment by curtiskphsc** [comment]
@Computerrat @C13L0 Ready for review

**Comment by C13L0** [comment]
@Computerrat Please review changes for accessibility and close ticket once verified.

**Comment by Computerrat** [comment]
Verified changes
